### PR TITLE
Express changeover

### PR DESCRIPTION
**Pull request by sifex**

### CODE REVIEW

**Comment by dcousens** [comment]
Love it

**Comment by sifex** [comment]
@dcousens Heroku costs $7 USD a month for always on. If it's the free tier, it takes 10 seconds to boot and respond to request. Any solution to Heroku that does the same thing for $5 USD a month given I'm hosting static websites?

**Comment by dcousens** [comment]
@sifex you're not going to find much cheaper than that generally,  but,  you could play with multiple sites on a box and scale it with elastic IPs instead

**Comment by sifex** [comment]
@dcousens I'll just see if I can find a CD like heroku but for static websites on my own server.

File: index.js [hunk 1]
```diff
@@ -20,9 +20,13 @@ includes.compile();
...
-  res.redirect(subdir + '/squad.xml');
+  res.redirect(subdir + '/xml/squad.xml');
 });
 
 router.listen(process.env.PORT || 3000, function () {
   console.log('Successfully Deployed')
 })
+
+app.use(function (req, res, next) {
+	res.status(404).send('Sorry cant find that!')
```
Comment (dcousens) [inline index.js]: Mate, that indentation.

[`npm install standard -g`](https://github.com/feross/standard),  and then to use it in your editor: https://github.com/feross/standard#sublime-text

Comment (dcousens) [inline index.js]: Also, depending on your middleware, you might want an error handling there too.
https://github.com/dcousens/easy-api/blob/master/index.js is a template I typically use.

**Comment by dcousens** [comment]
@sifex a CD?

**Comment by sifex** [comment]
Continuous Deployment

**Comment by dcousens** [comment]
git hooks

**Comment by sifex** [comment]
Yea, I did that for about a year
Sick of establishing one each for every website I did

So tempted to make a nice static CI service.

**Pull request closed without merging**